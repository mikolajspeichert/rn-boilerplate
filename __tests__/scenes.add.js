import 'react-native'
import React from 'react'
import configureStore from 'redux-mock-store'
import renderer from 'react-test-renderer'
// import { shallow } from 'enzyme'
import AddScreen from '@scenes/Add'

const state = {}
const mockStore = configureStore()

describe('AddScreen', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<AddScreen store={mockStore(state)} />)
    expect(wrapper).toMatchSnapshot()
  })
})
