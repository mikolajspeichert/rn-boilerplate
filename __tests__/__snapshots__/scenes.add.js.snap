// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddScreen renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#c764cc",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    [missing "en.add.welcome" translation]
  </Text>
</View>
`;
